frontend/client/Styles/screens/404.scss

```diff
@@ -1,17 +1,40 @@
 #not-found {
     display: flex;
+    justify-content: space-evenly;
+    // background-image: url('../../assets/fade-background-404.svg');
```

Comment:
Can any of these comments be deleted?

Reply:
Yes, absolutely.  Left those there by accident when experimenting w `linear-gradient` instead of svg file background.